recipes/default.rb

```diff
@@ -17,8 +17,8 @@
 # limitations under the License.
 node.default['yum']['powertools']['enabled'] = true
 node.default['yum']['powertools']['managed'] = true
```

Comment:
This should be replaced with using the `edit_resource` method now.

Reply:
I actually just forgot to remove this, powertools is installed by default in osl-repos bc it was in the base recipe